compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STOnFailCheckNode.java

```diff
@@ -0,0 +1,130 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Although this is an auto-generated code, we update the license header year manually and commit. This is a lacking requirement in treegen.